src/animation/universalTransition.ts

```diff
@@ -644,6 +765,7 @@ export function installUniversalTransition(registers: EChartsExtensionInstallReg
 
         // TODO multiple to multiple series.
         if (globalStore.oldSeries && params.updatedSeries && params.optionChanged) {
+            // TODO transitionOpt was used in an old implementation and can be removed now
```

Comment:
Hi, @pissang.

As you mentioned earlier that `transitionSeriesFromOpt` is an old implementation but not removed from code, should I remove related code?